ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-3node-ranGen.yaml

```diff
@@ -34,6 +34,11 @@ spec:
       metadata:
         name: openshift-node-performance-profile
       spec:
+        additionalKernelArgs:
+          - "idle=poll"
+          - "rcupdate.rcu_normal_after_boot=0"
+          - "kernel.timer_migration=1"
+          - "nohz_full=2-19,22-39" # same as .spec.cpu.isolated
```

Comment:
Since the source-crs version already has most of these, can we just set this last `nohz_full` line?  Or does setting part of a list in PGT reset the whole list @serngawy / @imiller0 ?

Reply:
Ya the whole list in the source-cr file would get replaced (it's not smart enough to append/prepend a list) if don't the include the whole thing.